content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,178 @@
+(manually using the Token).
+
+.. seealso::
+   :doc:`connect`
+
+Now the installation is complete, the devices connected to :abbr:`IoT (Internet of Things)` can be
+used to complete processes/actions.
+
+Troubleshooting
+===============
+
+.. _restart_windows_iot:
+
+Restart Windows IOT box
+-----------------------
+
+In some instances a manual restart of the physical :abbr:`IoT (Internet of Things)` box can resolve
+the issue of an :abbr:`IoT (Internet of Things)` box not showing up on the database. For the Windows
+virtual :abbr:`IoT (Internet of Things)` box a manual restart of the Odoo server can resolve
+database connection issues.
+
+To restart the virtual Windows IoT server:
+
+#. Type "Services" into the :guilabel:`Search Bar`
+#. Select the :menuselection:`Services` App and scroll down to the :guilabel:`Odoo` Service.
+#. Right click on :guilabel:`Odoo` and select :guilabel:`Start` or :guilabel:`Restart`. This action
+   will manually restart the Odoo IoT server.
+
+Firewalls
+---------
+
+The Windows virtual :abbr:`IoT (Internet of Things)` box software may not be reachable to the
+:abbr:`LAN (Local Area Network)` due to a firewall preventing the connection. Consult your local IT
+support to make exceptions (network discovery) in the :abbr:`OS (Operating System)` or firewall
+program. Windows has their own firewall as do other virus protection programs.
+
+.. example::
+   A client might encounter a time when they are able to reach the homepage of the :abbr:`IoT
+   (Internet of Things)` box, yet they cannot access it from another computer/mobile/tablet on the
+   same network.
+
+Making an exception on Windows Defender
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It's possible to allow other devices to access the Windows Virtual :abbr:`IoT (Internet of Things)`
+box while keeping the firewall on. This is done by creating a rule on *Windows Defender* and
+allowing communication through port `8069`. The following process describes the steps to take in
+order to make this exception.
+
+Navigating to Windows Defender to create the rule
+*************************************************
+
+First, open the *Windows Firewall* by navigating to the :guilabel:`Start Menu` and typing in
+`Firewall`. Then open the :menuselection:`Windows Defender Firewall` program. In the left menu,
+navigate to :guilabel:`Advanced Settings`.
+
+Following selecting :guilabel:`Advanced Settings`, select :guilabel:`Inbound Rules` in the left
+menu. Now in the right menu column, under :guilabel:`Inbound Rules`, and click on :guilabel:`New
+Rule`.
+
+Configure new rule
+******************
+
+On the :menuselection:`Rule Type` screen, select :guilabel:`Port`. Then click :guilabel:`Next`.
+Leave the rule application to :guilabel:`TCP` on the :menuselection:`Protocol and Ports` page.
+Select :guilabel:`Specific Local Ports` for :guilabel:`ports`. In the text box, type in `8069`.
```

Comment:
```suggestion
From the *Protocol and Ports* page, leave the rule application as :guilabel:`TCP`. Then,
select :guilabel:`Specific Local Ports` for the :guilabel:`ports` option. In the text box, type in `8069`.
```
Again, if my suggestion above for referencing forms/screens/pages is wrong, disregard that part of this change